### DISCUSSION TITLE
cancel teams and get refund

### DISCUSSION THREAD

**Discussion by dowo13** [community/community#103023] [New to GitHub] [answered]
i accidentally paid for github teams when i actually wanted to buy copilot. Can i cancel and get a refund for teams **and** then purchase copilot?

**Comment by queenofcorgis** [comment, marked as answer]
Hi @dowo13 ! The best place to go for answers to billing-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Reply by dowo13** [reply]
thanks - refunded
